typings/index.d.ts

```diff
@@ -752,6 +753,7 @@ declare module 'discord.js' {
 		public setRolePositions(rolePositions: RolePosition[]): Promise<Guild>;
 		public setSplash(splash: Base64Resolvable | null, reason?: string): Promise<Guild>;
 		public setSystemChannel(systemChannel: ChannelResolvable | null, reason?: string): Promise<Guild>;
+		public setSystemChannelFlags(systemChannelFlags: SystemChannelFlags, reason?: string): Promise<Guild>;
```

Comment:
The type for `systemChannelFlags` should be something similar to how Role#setPermissions has (`type PermissionResolvable = BitFieldResolvable<PermissionString>;` except adapted for this one)